frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -0,0 +1,255 @@
+import './_restore-pvc-modal.scss';
+
+import * as React from 'react';
+
+import { Form, FormGroup, Grid, GridItem, TextInput, Alert } from '@patternfly/react-core';
+import {
+  HandlePromiseProps,
+  ResourceIcon,
+  withHandlePromise,
+  validate,
+  resourceObjPath,
+  history,
+  RequestSizeInput,
+  Timestamp,
+  LoadingInline,
+} from '@console/internal/components/utils';
+import {
+  K8sResourceKind,
+  k8sCreate,
+  referenceFor,
+  VolumeSnapshotKind,
+  StorageClassResourceKind,
+  k8sGet,
+  PersistentVolumeClaimKind,
+} from '@console/internal/module/k8s';
+import {
+  ModalBody,
+  ModalComponentProps,
+  ModalSubmitFooter,
+  ModalTitle,
+  createModalLauncher,
+} from '@console/internal/components/factory';
+import {
+  NamespaceModel,
+  PersistentVolumeClaimModel,
+  VolumeSnapshotModel,
+  VolumeSnapshotClassModel,
+} from '@console/internal/models';
+import { getName, getNamespace, Status } from '@console/shared';
+import { StorageClassDropdown } from '@console/internal/components/utils/storage-class-dropdown';
+
+const dropdownUnits = {
```

Comment:
Note: Same object is available here - https://github.com/openshift/console/pull/5884/files#diff-3e19cb7f78e360efb73522185992f873R42. It needs be reused from file, which ever gets merge 1st.

Reply:
ack